llpc/lower/llpcSpirvLower.cpp

```diff
@@ -154,6 +154,25 @@ void SpirvLower::removeConstantExpr(Context *context, GlobalVariable *global) {
     replaceConstWithInsts(context, constVal);
 }
 
+// =====================================================================================================================
+// Replace global variable with another global variable
+//
+// @param global : Replaced global variable
+// @param replaceGlobalglobal : Replacing global variable
+void SpirvLower::replaceGlobal(GlobalVariable *global, GlobalVariable *replaceGlobal) {
+  removeConstantExpr(m_context, global);
+  for (User *user : global->users()) {
+    Instruction *inst = dyn_cast<Instruction>(user);
+    if (inst) {
+      m_builder->SetInsertPoint(inst);
+      Value *replacedValue = m_builder->CreateBitCast(replaceGlobal, global->getType());
+      user->replaceUsesOfWith(global, replacedValue);
+    }
+  }
+  global->dropAllReferences();
```

Comment:
What if the user if not an `Instruction`, e.g., other global variable or global alias?